kolibri/plugins/user_profile/viewsets.py

```diff
@@ -45,3 +45,28 @@ def get(self, request):
                 return Response({})
         except Exception as e:
             raise ValidationError(detail=str(e))
+
+
+class RemoteFacilityUserLoginViewset(APIView):
```

Comment:
I found this naming confusing - it made me think that this was an endpoint for logging into a remote device with a specific facility user, rather than getting a facility user while logged in? Could it be `RemoteFacilityUserAuthenticatedViewset`?

Also wonder if this should be a `POST` request to avoid putting the username and password in plain text in the URL?

Reply:
Happy to refactor the name. I hate naming things :) And for the `POST` recommendation, done too, it's clearly a good idea, thanks!